### PR TITLE
Test that RegExps aren't special-cased in Web IDL anymore

### PR DESCRIPTION
**Pull request by domenic**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/20753)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/303114934)

File: WebIDL/ecmascript-binding/no-regexp-special-casing.any.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+"use strict";
+// https://github.com/heycam/webidl/commit/bbb2bde">
```
Comment (tobie) [inline WebIDL/ecmascript-binding/no-regexp-special-casing.any.js]: remove the `">` at the end of the URL

**Review by tobie — APPROVED** [review]
This LGTM though I'd appreciate a tad more context in the initial comment. Something along the lines of: _"RegExps used to be special-cased in WebIDL, these tests check that this is no longer the case."_

**Comment by domenic** [comment]
I'd like to hold off merging until we find out why this didn't trigger some of Blink's remaining RegExp special casing. It may just be dead code in Blink, but it may be some obscure code path. https://bugs.chromium.org/p/chromium/issues/detail?id=662379#c7

**Pull request merged**